### PR TITLE
New doc: configure extensions private marketplace

### PR DESCRIPTION
**Pull request by Frezzle**
### Proposed changes

New page for documenting the new feature for configuring an extensions private marketplace.

The feature and doc is targeted at admin users with Docker Business subscription.

Screenshot 2023-11-14 at 11 30 33

### Related issues

https://docker.atlassian.net/browse/ECOPUB-269

### Where to look

Locally: http://localhost:1313/desktop/extensions/private-marketplace/
Deployed preview: https://deploy-preview-18667--docsdocker.netlify.app/desktop/extensions/private-marketplace/

### CODE REVIEW

File: content/desktop/extensions/private-marketplace.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
+---
+description: Extensions
+keywords: Docker Extensions, Docker Desktop, Linux, Mac, Windows,
+title: Configure an extensions private marketplace
+---
+
+This page shows administrators how to configure a private extensions marketplace with a curated list of extensions for your Docker Desktop users.
+
+Extensions private marketplace is designed specifically for organizations who don’t give developers root access to their machines.
```
Comment (gtardif) [inline content/desktop/extensions/private-marketplace.md:9]: I don't think we need to mention this. If user are root, the private marketplace will stil work and provide the curated list of extensinos, but user will have ways to bypass it and install extensions outside of the marketplace. If admins are ready to give root access to users, then that should not be an issue

**Review by gtardif — COMMENTED** [review]
Just a small nit, otherwise LGTM. 
We probably need to wait a bit before merging this since private marketplace is not a public feature yet

File: content/desktop/extensions/private-marketplace.md (hunk 1 above)
Comment (gtardif) [inline content/desktop/extensions/private-marketplace.md:9]:
```suggestion
Extensions private marketplace is designed for customers with Docker Business subscription.
```

**Comment by Frezzle** [comment]
Hey @gtardif, thanks for your review. My bad for not mentioning sooner, but this PR is being superseded by this one from @aevesdocker: https://github.com/docker/docs/pull/18681 since there were many suggested changes to make.

**Pull request closed without merging**